### PR TITLE
Bugfix/documentation

### PR DESCRIPTION
**Pull request by jeevanjoseph**
Fixes the issues identified in the review meeting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
+MuShop is a 3-tier web application that implements an e-commerce site. It is built to showcase the features of [Oracle Cloud Infrastructure (OCI)][oci]. This application is designed to run using only the Always Free tier resources. This repository contains the application code as well as the [Terraform][tf] code to create a [Resource Manager][orm] stack. This stack creates all the required resources and configures the application on the created resources.
 
 ## Getting Started
 
 The steps below guide you through deploying the application on your tenancy using the OCI Resource Manager.
 
-1. Download the latest MuShop [stack](./releases/mushop-basic-stack.zip)
+1. Download the latest [`mushop-basic-stack.zip`](./releases/mushop-basic-stack.zip) file.
 2. [Login](https://console.us-phoenix-1.oraclecloud.com/resourcemanager/stacks/create) to OCI to import the stack
```
Comment (peterj) [inline README.md]: OCI = Oracle Cloud Infrastructure

File: deploy/basic/README.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 ![MuShop Logo](../../images/logo.png)
 
-This is a Terraform configuration that deploys the MuShop Basic demo on [Oracle Cloud Infrastructure (OCI)][oci].
+This is a Terraform configuration that deploys the MuShop application demo on [Oracle Cloud Infrastructure (OCI)][oci].
```
Comment (peterj) [inline deploy/basic/README.md]: remove `demo`

File: deploy/basic/README.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
 
-MuShop Basic is a sample 3-tier web application that implements an e-commerce site. It is built to showcase the features of [Oracle Cloud Infrastructure (OCI)][oci]. This sample is designed to run using only the Always Free tier resources. This sample contains the application code as well as the [Terraform][tf] code to create a [Resource Manager][orm] stack. This stack creates all the required resources and configures the application on the created resources.
+MuShop is a 3-tier web application that implements an e-commerce site. 
+It is built to showcase the features of [Oracle Cloud Infrastructure (OCI)][oci].
+This application is designed to run using only the Always Free tier resources.
```
Comment (peterj) [inline deploy/basic/README.md]: Application is designed to run using only the Always Free tier resources. The repository contains the application code as well as the [Terraform][tf] code to create a [Resource Manager][orm] stack, that creates all the required resources and configures the application on the created resources.

File: deploy/basic/README.md (hunk 2 above)
Comment (junior) [reply deploy/basic/README.md]: And keep the basic notation, as will be different application soon.

Comment (jeevanjoseph) [reply deploy/basic/README.md]: Didnt we agree to keep the "application demo" like Hispter shop ?

Comment (jeevanjoseph) [reply deploy/basic/README.md]: MuShop basic sample application

File: deploy/basic/README.md (hunk 3 above)
Comment (jeevanjoseph) [reply deploy/basic/README.md]: done

**Pull request merged**